### PR TITLE
Fixed unable to create an instance of `AnotherExampleModel`.

### PR DESCRIPTION
**Pull request by HanlinMiao**
# Closes: #2526 
# What's Changed

- Added`get_absolute_url()` method to `AnotherExampleModel` class
- Added an `AnotherExampleModel` detail view template

# TODO

- [ ] Explanation of Change(s)
- [ ] Attached Screenshots, Payload Example
- [ ] Unit, Integration Tests
- [ ] Documentation Updates (when adding/changing features)
- [ ] Example Plugin Updates (when adding/changing features)
- [ ] Outline Remaining Work, Constraints from Design

### CODE REVIEW

**Review by glennmatthews — COMMENTED** [review]
A simpler fix might be to just change `AnotherExampleModelTable.name` from a `LinkColumn` to a `Column`?

**Comment by HanlinMiao** [comment]
> A simpler fix might be to just change `AnotherExampleModelTable.name` from a `LinkColumn` to a `Column`?

Is it intentional to leave out AnotherExampleModel detail view and by extension not linking the table Column?

I thought we missed it because I did see a `AnotherExampleModelView` present in [example_plugin.views.py](https://github.com/nautobot/nautobot/blob/f87ff3718e7060fa8e5f38820fd77c59e031692f/examples/example_plugin/example_plugin/views.py#L158)

**Comment by glennmatthews** [comment]
Oh that's interesting! Didn't know we had a detail view already defined for it.

File: changes/2526.fixed [hunk 1]
```diff
@@ -0,0 +1 @@
+Fixed unable to create an instance of `AnotherExampleModel` by adding `get_absolute_url()` method to `AnotherExampleModel` class and an `AnotherExampleModel` detail view template.
```
Comment (glennmatthews) [inline changes/2526.fixed:1]:
```suggestion
Fixed error in rendering the example plugin's `AnotherExampleModel` list view by adding a `get_absolute_url()` method to the `AnotherExampleModel` class and adding an `AnotherExampleModel` detail view template.
```

**Review by glennmatthews — APPROVED** [review]

**Pull request merged**